src/components/Layout/Page.js

```diff
@@ -25,6 +25,9 @@ const Page = ({ title, children }) => (
         background-image: url('${hexagonsLeft}'), url('${hexagonsRight}');
         background-repeat: repeat-y;
         background-position: top left, top right;
+
+        /* scale background with viewport */
+        background-size: calc(100px + (300 - 100) * ((100vw - 320px) / (1920 - 320)));
```

Comment:
clever!

Reply:
Yeah it's a cool technique based on the fluid type algorithm :slightly_smiling_face:

```
calc([minimum size] + ([maximum size] - [minimum size]) * ((100vw - [minimum viewport width]) / ([maximum viewport width] - [minimum viewport width])));
```

https://css-tricks.com/snippets/css/fluid-typography

Reply:
Could you comment that in the component please?

Reply:
Also, fyi, `min`/`max` and `clamp` will make this way easier in the future :slightly_smiling_face:
https://css-tricks.com/simplified-fluid-typography